tests/utils/test_modeling_utils.py

```diff
@@ -815,6 +815,62 @@ def test_checkpoint_variant_local_sharded_safe(self):
         for p1, p2 in zip(model.parameters(), new_model.parameters()):
             self.assertTrue(torch.allclose(p1, p2))
 
+    def test_checkpoint_safetensors(self):
```

Comment:
I think it would be good (to all of us) that we write a short comment about why we add a test.

As I am sure we already have some tests that are about saving/loading. And when we add these 2 new tests, the main test case is about `failing the loading if a format is specified (as we don't switch to other format)`.


Reading the test names here doesn't tell me what is the main objective, and I will be confused why we still add new saving/loading tests.